scss/bootstrap/components/_editable-content.scss

```diff
@@ -0,0 +1,31 @@
+.crm-editable-enabled:not(.crm-editable-editing):hover {  
```

Comment:
@AkA84 crm-editable-enabled comes from modules/civicrm/js/jquery/jquery.crmEditable.js and is generated for a section which is having enabled editing.